src/annotator/sidebar.js

```diff
@@ -123,7 +134,7 @@ export default class Sidebar extends Guest {
     }
 
     if (this.plugins.BucketBar) {
-      this.plugins.BucketBar.element.addEventListener('click', () =>
+      this._registerEvent(this.plugins.BucketBar.element, 'click', () =>
```

Comment:
It's not reported here, but this line is a test miss (you can see it in local test coverage information; let me know if you need any help finding that!).

Reply:
Thanks, I will add a test for that.

FYI, I tried to add a test for src/annotator/sidebar.js#L260 and I couldn't. It was extremely difficult to trigger hammer.js event.